deployment-scripts/update-manifest.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
+if [[ -z "$1" ]] || [[ -z "$2" ]]  || [[ -z "$3" ]]; then 
+  echo "Usage: update-manifest.sh <manifest-git-repo> <component-name> <image-uri>"
+  exit -1;
+fi
+
+
+git_url=$1
+comp=$2
+uri=$3
+
+set -euo pipefail
+
+if [[ ! -z "${GIT_DEPLOYMENT_KEY+x}" ]]; then 
+  git-set-creds-github.sh "$GIT_DEPLOYMENT_KEY"
+fi
+
+cd $DIR
+repo_name=$(echo $git_url| sed -e 's#.*/## ; s/.git//g')
+rm -rf $repo_name
+git clone --depth 1 --single-branch --branch master $git_url
+if [[ $? -ne 0 ]]; then 
+  echo "Failed to clone repo"
+  exit -1
+fi
+cd $DIR/$repo_name
+git pull
+git checkout -B "robot-${comp}"
+
+echo $uri > manifest/$comp
```

Comment:
Something for the future but have you thought about just updating the Docker tag rather than the entire URI? It seems safer to me.

Reply:
We did that way because it copes with creation of a new service as well as updating it. I suggest we leave this as it is for now

Reply:
Okay.